testdata/book/include_vars_included.yml

```
@@ -0,0 +1,21 @@
+desc: Include vars (included)
+if: included
+steps:
+  -
+    test: |
+      // If it is a primitive type, the value is passed as it is.
+      vars.a == 12345
+      && vars.b == "54321"
+      // If there is a variable expansion, it is reinterpreted in a form without quotes.
+      && vars.c == 12345
+      && vars.d == 54321
+      && vars.e == 123459
+      && vars.f == 543219
+      && vars.g == 912345
+      && vars.h == 954321
```

The following expectations were assumed
```suggestion
      // If there is a variable expansion, it is reinterpreted in a form without quotes.
      && vars.c == 12345
      && vars.d == "54321"
      && vars.e == "123459"
      && vars.f == "543219"
      && vars.g == "912345"
      && vars.h == "954321"
```

---

Then, if there is variable expansion, there is no way to pass numbers.